Formula/gajim.rb

```diff
@@ -0,0 +1,192 @@
+class Gajim < Formula
+  include Language::Python::Virtualenv
+
+  desc "Full featured and easy to use XMPP client"
+  homepage "https://gajim.org/"
+  url "https://gajim.org/downloads/1.1/gajim-1.1.2.tar.bz2"
+  sha256 "3283ea5239f67d84874d4efb5b0b0780d1120e5b46c37a5627e5a3121f1ba0d3"
+
+  depends_on "adwaita-icon-theme"
+  depends_on "freetype"
+  depends_on "gettext"
+  depends_on "gtk+3"
+  depends_on "jpeg"
+  depends_on "librsvg"
+  depends_on "pygobject3"
+  depends_on "python"
+
```

Comment:
Missing dependencies:
```suggestion
  depends_on "glib-networking"
  depends_on "gnupg"
  depends_on "gspell" => :optional
  depends_on "gupnp"
  depends_on "hicolor-icon-theme"
  depends_on "hunspell" => :optional
  depends_on "libsecret"
  depends_on "openjpeg" # Pillow
```